roles/seqreports/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+
+- name: create conda environment for seqreports
+  shell: "{{ anaconda_path }}/bin/conda create -n {{ seqreports_venv_name }} -c anaconda python=3.8 -y"
+  args:
+    creates: "{{ anaconda_path }}/envs/{{ seqreports_venv_name }}/bin/python3"
+
+- name: Set host fact boolean for miarka deployment
+  set_fact:
+    apply_to_conda: "{{ apply_to_conda | default(true) }}"
+
+- name: ensure seqreports data path exists
+  file:
+    path: "{{ seqreports_data_path }}"
+    state: directory
+
+- name: ensure nextflow directories exist
+  file:
+    path: "{{ item }}"
+    state: directory
+  loop:
+    - "{{ seqreports_nxf_temp }}"
+    - "{{ seqreports_nxf_work }}"
+    - "{{ seqreports_nxf_logs }}"
+
+- import_tasks: pre_reqs.yml
+
+- import_tasks: seqreports_dev.yml
+  when: not apply_to_conda | bool
+
+- name: Update and deploy checkQC config
+  include_role:
+    name: checkqc
+    tasks_from: deploy_config.yml
+
```

Comment:
Hm, this won't work when all tasks in checkqc are in place. The alternative is to put deploy_config in its own role. We have discussed this matter (Matilda and I), and concluded that it would be such a small role. However, we were not really satisfied with this solution either. If you think we should put it in a separate role, we'll do it.

Reply:
Good to know of the nextflow dependency. :+1:

Reply:
ok but if the only purpose of the checkqc stuff is to copy the config, how about just including it as a Jinja template with the `bcl2fastq_output_path` as a variable? 
similar to e.g. https://github.com/NationalGenomicsInfrastructure/miarka-provision/blob/devel/roles/func_accounts/templates/supervisord_site.conf.j2

Reply:
I'm a little bit concerned about that checkqc.config might change in future, within the checkqc repo. If so, the version within ansible will be out of date. Is this an issue? What do you think of this @matrulda?

Reply:
I see. If I understand correctly, the purpose of the checkqc config is to parse the quality thresholds we use. In the near future, checkQC should also be deployed on Miarka so then this should not be an issue anyway.

However, would you be ok with (for now) moving the code to this role? I think it is a little bit confusing to just have a very specific and small task as a stand-alone role. Then once checkqc is deployed on Miarka it will have a proper role and then seqreports just needs to point to the config deployed by checkqc.

Reply:
ok, i move it!